java/code/src/com/redhat/rhn/domain/server/ServerFQDN.java

```diff
@@ -73,6 +74,20 @@ public void setName(String nameIn) {
         this.name = nameIn;
     }
 
+    /**
+     * @return primary String which indicates primary interface
+     */
+    public String getPrimary() {
```

Comment:
Could we turn those accessors into `boolean` ones? I find the "YN" string confusing.

See how this is done, for example, in the PAYG attribute of Server:

https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/domain/server/Server_legacyUser.hbm.xml#L35

https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/domain/server/Server.java#L130

https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/domain/server/Server.java#L2079-L2093

My understanding was that the "magic" `yes_no` Hibernate column type hides away all string handling - or at least that would be my hope :angel: